bot/seasons/evergreen/TriviaQuiz.py

```diff
@@ -0,0 +1,263 @@
+            return
+        if message.author.bot:
+            return
+
+        game = self.games[message.channel.id]
+        question_data = game.question
+        answer = question_data["answer"].lower()
+        user_answer = message.content.lower()
+        ratio = fuzz.ratio(answer, user_answer)
+        if ratio > 84:
+            points = question_data["points"] - game.hints*25
+            if message.author in game.players:
+                author_index = game.players.index(message.author)
+                game.points[author_index] = game.points[author_index] + points
+            else:
+                game.players.append(message.author)
+                game.points.append(points)
+
+            await channel.send(f"{message.author.mention} got it right! Good job :tada:"
+                               f"You got {points} points.")
+            await self.score_embed(channel)
+            await self.send_question(channel)
+        elif ratio in range(75, 84):
+            await channel.send(f"Your close to the answer {message.author.mention}")
+
+    async def send_hint(self, channel, question_dict):
+        """Function to be called whenever a hint has to be sent."""
+        await asyncio.sleep(10)
+        try:
+            game = self.games[channel.id]
+        except KeyError:
+            return
+
+        # Checking if the question is the same after 10 seconds.
+        if question_dict["id"] == game.question["id"]:
+
+            # If the max number of hints is already reached, then send the answer.
+            if 2 - game.hints == 0:
+                return await self.send_answer(channel)
+
+            hint_list = question_dict["hints"]
+            hint_index = game.hints
+            hint = hint_list[hint_index]
+            game.hints += 1
+            message = f"**Hint {game.hints}**: {hint}\n*Number of hints remaining: {2-game.hints}*"
+            await channel.send(message)
+            await self.send_hint(channel, question_dict)
+        else:
+            pass
+
+    async def send_answer(self, channel):
+        """A function to send the answer in the channel if no user have given the correct answer even after 2 hints."""
+        game = self.games[channel.id]
+        answer = game.question["answer"]
+        response = random.choice(wrong_ans_responses)
+        expression = random.choice(annoyed_expressions)
+        await channel.send(f"{response} {expression}, the correct answer is **{answer}**.")
+        self.games[channel.id].unanswered_questions += 1
+        await self.score_embed(channel)
+        await self.send_question(channel)
+
+    @tquiz.command(name="score")
+    async def send_score(self, ctx):
+        """Show scoreboard of the game running in this channel."""
+        await self.score_embed(ctx.channel)
+
+    async def score_embed(self, channel):
+        """Show score of each player in the quiz."""
+        if channel.id not in list(self.games.keys()):
```

Comment:
There shouldn't be a need to create a list out of dictionary keys.
```suggestion
        if channel.id not in self.games:
```